sources/parsers/psmt2_to_alt_ergo.ml

```diff
@@ -356,10 +356,10 @@ module Translate = struct
     | Cmd_Assert(assert_term) ->
       (translate_assert (pos command) assert_term) :: acc
     | Cmd_CheckEntailment(assert_term) ->
-      Options.set_unsat_mode false;
+      Options.set_output_mode Native;
```

Comment:
I've made an error here, this should not set the output to Native. Could you change the code so that the Output mode should be done when we analyse the extension of the input file.
By default :
- .smt2 | .psmt2 -> Smtlib
- .why | .mlw | .ae -> Native